src/classes/HH_NameSpec_TEST.cls

```diff
@@ -54,13 +54,13 @@ private with sharing class HH_NameSpec_TEST {
     static testMethod void OneContact() {
         if (strTestOnly != '*' && strTestOnly != 'OneContact') return;
         
-        list<Contact> listCon = new list<Contact>();
-        Contact c1 = new Contact (
-            Salutation = 'Skipper',
-            FirstName = 'John',
-            LastName = 'Doe'
-        );
-        listCon.add(c1);
+        Contact[] listCon = new Contact[] {
```

Comment:
These are tests that do not need individual contacts declared. Even though c1 and listCon[0] point to the same contact for example, it is easier to maintain and read the code if we add items to the list when it is constructed. It also reduces number of lines in the code.